edgemesh/tools/initContainer/script/edgemesh-iptables.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/bin/bash
```

Comment:
`#!/usr/bin/env bash`  seems better, which searches PATH for bash, and bash is not always in /bin, particularly on non-Linux systems. Thought? Xref:  https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash

Reply:
yes, "#!/usr/bin/env bash" is better